lib/entities/quote/DutchQuote.ts

```diff
@@ -68,7 +69,13 @@ export class DutchQuote implements IQuote {
   public static amountInImprovementExactOut = BigNumber.from(9999);
 
   // build a dutch quote from an RFQ response
-  public static fromResponseBody(request: DutchRequest, body: DutchQuoteJSON, nonce?: string): DutchQuote {
+  public static fromResponseBody(
+    request: DutchRequest,
+    body: DutchQuoteJSON,
+    nonce?: string,
+    portionBips?: number,
```

Comment:
thoughts on putting `portionBips` and `portionRecipient` within `DutchQuoteJSON`? then the method signature can stay the same.